File: src/main/java/org/example/Domain/Echipa.java
package org.example.Domain;

public class Echipa extends Entity<Long>{
    String nume;

    public Echipa(Long id,String nume) {
        super.id=id;
        this.nume = nume;
    }

    public String getNume() {
        return nume;
    }

    public void setNume(String nume) {
        this.nume = nume;
    }
}

File: Domain/Echipa.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Motociclete.Domain
{
    public class Echipa : Entity<long>
    {
        public string nume {  get; set; }
        public Echipa(long id, string nume) {
            this.id = id;
            this.nume = nume;
        }
    }
}
